src/main/resources/db/migration/1_createschema.xml

```diff
@@ -44,6 +42,8 @@
             <column name="valid_to" type="varchar(20)"/>
         </createTable>
         <addNotNullConstraint columnDataType="BIGINT(20)" columnName="id" tableName="product" />
+        <addNotNullConstraint columnDataType="varchar(50)" columnName="name" tableName="product" />
+        <addPrimaryKey tableName="product" columnNames="id, name" constraintName="pk_product"/>
```

Comment:
Please revert this change. If you want to make some attiribute unique, just add unique constraint. After change primary key to group of attributes you allowed following behavior:
id name
1  a
1  b
2  a
It looks strange, isn't it?)))

Reply:
Agree, my mistake, changed to unique key